projects/make-a-vis/src/lib/mav-selection/graphic-variable-type/graphic-variable-type.component.ts

```diff
@@ -320,7 +365,7 @@ export class GraphicVariableTypeComponent implements OnChanges {
     this.requiredGraphicVariablesMapping.forEach((reqGVIds, gsoId) => {
       result = result && reqGVIds.every((gvId) => {
         if (this.selectedDataVariablesMapping.get(gsoId)) {
-          return this.selectedDataVariablesMapping.get(gsoId).get(gvId) ? true : false;
+          return Boolean(this.selectedDataVariablesMapping.get(gsoId).get(gvId));
```

Comment:
`return !!this.selectedDataVariablesMapping.get(gsoId).get(gvId)` would be more concise.